### DISCUSSION TITLE
resource managed about Database-oriented multi-tenancy: 面向数据库的多租户的资源管理

### DISCUSSION THREAD

**Discussion by Jz-Lin** [milvus-io/milvus#28650] [Q&A and General discussion]
关于面向数据库的多租户管理，怎么实现资源的管理呢。实现数据库级别的资源管理。有没有相关的介绍文档。
Regarding database-oriented multi-tenant management, how to implement resource management and achieve database-level resource management. Is there any relevant introductory document?

**Comment by yhmo** [comment]
https://milvus.io/docs/users_and_roles.md
RBAC的权限设置目前主要只有Collection级别和Global级别，没有针对database的设置，我觉得目前没法做到。

**Reply by jaime0815** [reply]
https://milvus.io/docs/manage_databases.md 有rbac 操作db 的说明

**Comment by zhuwenxing** [comment]
你说的资源隔离是不是类似于这样的东西？
https://milvus.io/docs/resource_group.md

**Reply by yhmo** [reply]
resource group是针对query node和collection replica的隔离，就是指定哪个collection由哪个query node加载。至于这个collection是谁使用它不关心，这个collection是属于哪个database它也不关心。

**Reply by Jz-Lin** [reply]
嗯嗯  这个是针对collection  级别的，我想是说 有没有database 级别的，想要达到比如不同的业务场景使用不同的database 实现隔离，根据业务量和重要程度进行资源的分配，这样的话 应该需要database 级别的资源隔离。

**Reply by yhmo** [reply]
@Jz-Lin 
你看上面那个链接https://milvus.io/docs/manage_databases.md 是否能满足你的要求
